jina/excepts.py

```diff
@@ -90,3 +90,7 @@ class GRPCServerError(Exception):
 
 class GatewayPartialMessage(Exception):
     """Gateway receives a multi-part message but it can not handle it"""
+
+
+class MilvusDBException(Exception):
+    """Any time Milvus client returns Status not OK"""
```

Comment:
What about having `ExecutorError` and inherit `MilvusDBException` or `ExternalServiceError`. I wanted to make clear that the problem comes from Milvus so that it can be more informative.